opwen_email_server/swagger/email-receive.yaml

```diff
@@ -36,6 +36,35 @@ paths:
         403:
           description: Request from unregistered client.
 
+  /echo:
+
+    post:
+      operationId: opwen_email_server.integration.connexion.service_request_email_receive
+      summary: Webhook listening for request for services emails via Sendgrid.
+      consumes:
+        - multipart/form-data
+      parameters:
+        - $ref: '#/parameters/ClientId'
```

Comment:
This parameters list is repeated with the other endpoint and it's a rather long duplication which means it's easy in the future to make a change to one list and miss it in the other. I wonder if Connexion supports [YAML anchors](https://medium.com/@kinghuang/docker-compose-anchors-aliases-extensions-a1e4105d70bd) so that we could remove the code duplication.

Reply:
Oh interesting! I'll look into it!